patches/minecraft/net/minecraft/world/level/block/entity/BlockEntity.java.patch

```diff
@@ -9,12 +9,12 @@
     private static final Logger f_58854_ = LogUtils.getLogger();
     private final BlockEntityType<?> f_58855_;
     @Nullable
-@@ -23,11 +_,14 @@
+@@ -23,11 +_,13 @@
     protected final BlockPos f_58858_;
     protected boolean f_58859_;
     private BlockState f_58856_;
+-
```

Comment:
This patch change looks wrong to me, looks like this PR added a trivial whitespace change to the patch

Reply:
Agreed. Please remove it so we can keep the patch surface as low as possible.